docs/src/documentation/03-components/05-input/inputfield/03-accessibility.mdx

```diff
@@ -51,7 +110,7 @@ In this example, both the "email-hint" text and the error message will be announ
 
 When using InputField within an InputGroup, the `aria-describedby` association follows these rules:
```

Comment:
Maybe I'm missing some piece of the puzzle, but it's not clear to me how `aria-describedby` is related to the examples below, where no ariaDescribedby is set. 🤔

Reply:
How is this sentence confusing, I don't see it, can you please explain and suggest what would you write there?

I can see how text in example can be confusing though, would it help to rewrite that one?

In `Group-level error/help messages`, rewrite "If the InputGroup has `error`/`help` messages, these will be properly associated with all child components:" to "If the InputGroup has `error`/`help` messages, all child components will have `aria-describedby` set to these values:"

Reply:
> In Group-level error/help messages, rewrite "If the InputGroup has error/help messages, these will be properly associated with all child components:" to "If the InputGroup has error/help messages, all child components will have aria-describedby set to these values:"

yes, that's what I meant. IMO, it was not mentioned how aria-describedby is connected with error/help messages. Adding `If the InputGroup has error/help messages, all child components will have aria-describedby set to these values:` makes it more understandable. Or `If the InputGroup has error/help messages, all child components will have aria-describedby set to these values **by default:**` also could be ok, I guess.

Reply:
> it was not mentioned how aria-describedby is connected with error/help messages

I see. This is more explained in the Automatic Accessibility Features - do you think it's enough like this?